index.js

```diff
@@ -26,6 +26,21 @@ app.get('/api/reviews/products/:productId',
     .then((result) => res.send(result))
     .catch((error) => res.send(error)));
 
+app.post('/api/reviews/products/:productId',
```

Comment:
@dbooyah93 Just to confirm intention, your post request isn't actually posting anything to the database, is that correct?